lib/chutney/TorNet.py

```diff
@@ -1294,13 +1304,11 @@ def getNodePublishedDirInfoPaths(self):
 
     def getNodeDirInfoStatusPattern(self, dir_format):
         """Returns a regular expression pattern for finding this node's entry
-           in a dir_format file.
-
-           The microdesc format is not yet implemented, because it requires
-           the ed25519 key. (Or RSA block matching, which is hard.)
+           in a dir_format file and returning None if nickname or ed25519_id key not found.
```

Comment:
A few fixes:
* nickname is never checked for None, because it's a required variable.  So it never causes this function to return None.
* there are two different ways that this function can return None
* we want to explain why the function returns None, and what that means. We don't need to repeat exactly what the code says.
```suggestion
           in a dir_format file. Returns None if the requested pattern is not available.
```